spec/intl.html

```diff
@@ -1345,7 +1345,7 @@ <h1>CalendarEraYear ( _calendar_, _dateLike_ )</h1>
           <h1>
             CalendarDateToISO (
               _calendar_: a String,
-              _fields_: a Record,
+              _fields_: an Object with the own data properties listed in <emu-xref href="#table-temporal-field-requirements"></emu-xref>,
```

Comment:
does this restrict the [[Prototype]] as well, and also disallow a Proxy?

Reply:
Should it say "an ordinary Object with..."?

Reply:
That seems good